WordPress/src/main/java/org/wordpress/android/util/DisplayUtilsWrapper.kt

```diff
@@ -8,14 +8,15 @@ import javax.inject.Inject
 class DisplayUtilsWrapper @Inject constructor(private val contextProvider: ContextProvider) {
     fun getDisplayPixelWidth() = DisplayUtils.getDisplayPixelWidth()
 
-    fun isLandscapeBySize() = getDisplayPixelWidth() > DisplayUtils.getDisplayPixelHeight(contextProvider.getContext())
+    fun isLandscapeBySize() =
+            getDisplayPixelWidth() > DisplayUtils.getWindowPixelHeight(contextProvider.getContext())
 
     fun isLandscape() = DisplayUtils.isLandscape(contextProvider.getContext())
 
     fun isTablet() = DisplayUtils.isTablet(contextProvider.getContext()) ||
             DisplayUtils.isXLargeTablet(contextProvider.getContext())
 
-    fun getDisplayPixelHeight() = DisplayUtils.getDisplayPixelHeight(contextProvider.getContext())
+    fun getDisplayPixelHeight() = DisplayUtils.getWindowPixelHeight(contextProvider.getContext())
```

Comment:
Can we rename `getDisplayPixelHeight()` to `getWindowPixelHeight()`?

Reply:
Yup right. Done in [b9684d4](https://github.com/wordpress-mobile/WordPress-Android/pull/16677/commits/b9684d4ac4887bde4c7b93bd03dccf9544ff5fa9)